libcontainerd/errors.go

```diff
@@ -1,46 +1,13 @@
 package libcontainerd
 
-import "errors"
+import (
+	"errors"
 
-type liberr struct {
-	err error
-}
+	"github.com/docker/docker/errdefs"
+)
 
-func (e liberr) Error() string {
-	return e.err.Error()
-}
+func newNotFoundError(err string) error { return errdefs.NotFound(errors.New(err)) }
```

Comment:
For a follow-up, we should get rid of these helpers (now it's helpers using helpers :joy:)